SdtReactApi/ClientApp/src/Pages/Leaves/LeaveBalance/LeaveBalance.js

```
@@ -0,0 +1,204 @@
+                        }
+                        newNotifications.push({ message: messages[i].reason, success: false });
+                    }
+                    setNotification([...notification, ...newNotifications]);
+                }
+            })
+    }
+    const handleDelete = (id) => {
+        axios.post(`api/leave/DeleteLeaveBalance`, {
+            id: id,
+            userId: 1,
+            leavetype: '',
+            available: 1,
+            userName: ''
+        }, {
+            headers: {
+                'Authorization': `${auth.accessToken}`
+            }
+        })
+            .then(response => {
+                handleGetEmployeeLeaveBalance()
+                toast.success(response?.data?.message[0]?.reason, { position: "bottom-center", theme: "dark" });
+                setNotification([...notification, { message: response?.data?.message[0]?.reason, success: true }])
+            })
+            .catch(error => {
+                toast.error(error?.response?.data?.message[0]?.reason, { position: "bottom-center", theme: "dark" });
+                setNotification([...notification, { message: error?.response?.data?.message[0]?.reason, success: false }])
+            })
+    }
+    const handleGetLeaveTypes = () => {
+        axios.get('api/leave/GetLeaveTypes', {
+            headers: {
+                'Authorization': `${auth.accessToken}`
+            }
+        })
+            .then(response => {
+                setLeaveTypes(response?.data?.requestedObject)
+            })
+            .catch(error => {
+            })
+    }
+    const handleGetEmployeeDetails = () => {
+        axios.get(`api/leave/GetEmployeeDetails/${auth.userId}`, {
+            headers: {
+                'Authorization': `${auth.accessToken}`
+            }
+        })
+            .then(response => {
+                setEmployeeDetails(response?.data?.requestedObject)
+            })
+            .catch(error => {
+            })
```

why error is not handled